cmd/vc-rest/startcmd/start.go

```diff
@@ -299,6 +297,14 @@ func buildEchoHandler(conf *Configuration, cmd *cobra.Command) (*echo.Echo, erro
 		StateStore:              oidc4StateStore,
 		IssuerInteractionClient: issuerInteractionClient,
 		IssuerVCSPublicHost:     conf.StartupParameters.hostURLExternal,
+		DefaultHTTPClient:       httpClient,
+		PreAuthorizeClient: func() *http.Client {
+			client := getDefaultHttpClient(tlsConfig)
+			client.CheckRedirect = func(req *http.Request, via []*http.Request) error {
```

Comment:
do we really want to disable following redirects? plz elaborate on this...

one more issue - it's not clear what is the purpose of `getDefaultHttpClient` function? It returns client by reference so any changes to this instance will affect it in other places as well.

Reply:
getDefaultHttpClient  - always returns a new reference to a new client. 
we need to disable redirect, as we don't need it for auto pre-auth flow https://github.com/trustbloc/vcs/pull/938/files#diff-6e396ac797089209a3155761638fd798eefc3e9c1d780e9019d8793e33e07371R409